engine/swarm/services.md

```diff
@@ -70,6 +70,107 @@ $ docker service create --name helloworld alpine ping docker.com
 9uk4639qpg7npwf3fn2aasksr
 ```
 
+### Specify the image version the service should use
+
+When you create a service without specifying any details about the version of
+the image to use, the service uses the latest version available in Docker Hub
+or your registry. You can force the service to use a specific version of
+the image in a few different ways, depending on your desired outcome.
+
+When you don't specify a tag, you get digest that corresponds to the `latest`
+tag of an image, by convention. There is no guarantee that the version tagged
+with `latest` is actually the newest or most up-to-date version. In general, it
+is recommended that you choose a specific version of an image rather than
+relying on `latest`. Otherwise, service tasks that deploy at different times may
+use different versions of the image, and this may not be what you intend.
```

Comment:
Did this text change from before?  Sorry if I missed this earlier, but the messaging is confusing here. It's good to be clear that relying on `latest` isn't a good idea, but `service tasks that deploy at different times may use different versions of the image` is exactly what tag->digest resolution is meant to prevent. The only time this can fail is when the registry is unreachable or v1. Maybe it makes sense to specify that?

Especially since this is described below, so just a pointer to that might suffice.